.github/workflows/dev.yml

```diff
@@ -8,7 +8,10 @@ on:
 
 jobs:
   build:
-    uses: axonivy-market/github-workflows/.github/workflows/dev.yml@v2
+    uses: ./.github/workflows/ci-fork.yml
```

Comment:
once the common pipeline is upgraded and tagged, we can turn this 'usage' back to the official pipeline and remove the `ci-fork.yml`again